tendrl/node_agent/objects/node/atoms/cmd.py

```diff
@@ -7,6 +8,6 @@
 class Cmd(object):
     def run(self, parameters):
         cmd = parameters.get("Node.cmd_str")
-        cmd = ["nohup"] + cmd.split(" ")
+        cmd = ["nohup"] + shlex.split(cmd)
```

Comment:
As part of https://github.com/Tendrl/common/pull/79 command executor comes to common module. We might need refactor later to use the same.